backend/clubs/migrations/0001_initial.py

```diff
@@ -56,6 +56,7 @@ class Migration(migrations.Migration):
                     ),
                 ),
                 ("accepting_members", models.BooleanField(default=False)),
+                ("enables_subscription", models.BooleanField(default=False)),
```

Comment:
In this case, the process succeeds, but since you initially specified `default=False`, you'll find that all clubs that are currently in the database have their subscriptions turned off.

Either way, best to never modify previous migrations files that are already in production.